activities.json

```
@@ -35,6 +35,18 @@
     "title": "BigInt",
     "url": "https://tc39.github.io/proposal-bigint"
   },
+  {
+    "ciuName": null,
+    "description": "This proposal adds an API for prompting and querying the user’s contacts for one or more items with a handful of contact properties.",
+    "id": "contact-picker",
+    "mozBugUrl": null,
+    "mozPosition": "defer",
+    "mozPositionDetail": "This API innnovates in some ways beyond several previous Contacts APIs, though leaves their source unspecified, and uses different properties than HTML autofill field names.",
```

The current text states facts about the API, but don't really relate to the position. How about: 


```suggestion
    "mozPositionDetail": "Although we see this API as a non-harmful addition to the Web, Mozilla defers working directly on this API at this time. We encourage the WICG community to continue to iterate on a data model that is well-suited for representing contacts in a manner that is both privacy preserving and OS independent.",
```

---

Two quick comments -- (1) My understanding is that "defer" indicates that we are deferring _stating a position_ pending additional data or other developments. Your proposal takes it to mean something subtly different (deferring standards work, which is related but not exactly the same thing). (2) While I understand where you're coming from with the phrasing "working directly on this API," it's easy to see how external parties my read "working" as "implementing" rather than "contributing to its specification." I believe phrasing like "...working directly on this API specification..." would make this less likely.

---

Good suggestion re: 2.. and also good points about 1. "Defer" is indeed, "Mozilla does not intend to look at this specification at all in the near future." I guess I got confused because we did look at it, but now we won't be investing any more time looking at it at all. I'm indeed biased as WICG Chair, as I do duck in an out of a lot of WICG specs as part of my day to day work.

---

Yeah, I think Marcos's suggestion doesn't fit with a "defer" position.  I'd be OK with the part about moving it into the description, though.  That would leave the "detail" part *almost* blank -- though Tantek has a "leaving the source unspecified" that Marcos didn't copy, and I'm also not sure what that means.  @tantek should we drop or move that part as well?

---

> Yeah, I think Marcos's suggestion doesn't fit with a "defer" position. 

Re-checked my assumptions there. We should maybe clarify that a bit in the README where "defer" is defined... if I got it wrong, then likely others internally and externally folks are interpreting it incorrectly. 

There still needs to be something that says "we think this is in good hands, even though we don't have the resources to participate right now." Or is that not a thing? 

> Tantek has a "leaves their source unspecified" .. I'm also not sure what that means

I wasn't sure either... I was guessing that means the OS or app (e.g., Contact.app)? Revealing that would be weird tho. @tantek?